test/ui-test/testSuites/suite_status/shared/steps/chatSteps.py

```diff
@@ -73,6 +73,15 @@ def step(context, message):
 def step(context, message_index, message):
     _statusChat.reply_to_message_at_index(message_index, message)
     _statusChat.verify_last_message_sent(message)
+    
+@When("the user can edit the message at index |any| and change text to \"|any|\"" )
+def step(context, message_index, message):
+    _statusChat.edit_message_at_index(message_index, message)
+    _statusChat.verify_last_message_sent(message)
+
+@Then("the user can read the message marks as \"|any|\"")
+def step(context, message):
+    _statusChat.verify_last_message_sent(message)
```

Comment:
But does this validation verify the `(Edited)` mark :thinking: ??

Reply:
yes :)